integration_tests/models/materializations/test_insert_by_period.sql

```diff
@@ -0,0 +1,16 @@
+{{
```

Comment:
Can we do something hacky to get these tests to pass? Let's just not configure this model for Postgres, making it use the `view` materialization instead

```suggestion
{% if target.type != 'postgres' %}
  {{
  ...
  }}
{% endif %}
```

Reply:
^ I guess this fails during parsing? Boo. Might be worth just implementing that `if/else` block

Reply:
It turns out that the circlei script doesn't run the makefile, I just made the circle config match.